File: projects/m1/007-day-old-bread/solutions/DomenicoTraina/day-old-bread.py
bread_price= 3.49
discount_bread_price=3.49*0.4 
number_old_breads = int (input ("how many pieces of old bread? "))

full_price=      bread_price*number_old_breads
total_discount=  discount_bread_price*number_old_breads
must_pay=        full_price-total_discount

print ("Price of bread is:            {} $".format(bread_price))
print (f"Full Price of old bread is:  {full_price:.2f} $")
print (f"Discount for {number_old_breads} pieces is:    {total_discount:.2f} $")
print (f"You must pay only:           {must_pay:.2f} $")

File: projects/m1/007-day-old-bread/solutions/alessiacaporaso/day-old-bread.js
const breadPrice = 3.49;
const discount = (breadPrice * 60) / 100;
// number of pieces of bread
let numberBreadUser = parseInt(prompt('How many pieces do you want to buy?'));
//price bread without discount
const priceBreadBought =  (numberBreadUser * breadPrice).toFixed(2);
// discount old bread
const discountOldBread = (discount * numberBreadUser).toFixed(2);
// tot price
const totPrice = (priceBreadBought - discountOldBread).toFixed(2); 
alert(`The price of bread is:\n${priceBreadBought}\nDiscount is:\n${discountOldBread}\nPrice total is:\n${totPrice}`)